paddlets/datasets/tsdataset.py

```diff
@@ -1963,7 +1963,7 @@ def freq(self):
         return self._freq
 
     @classmethod
-    def concat(cls, tss: List["TSDataset"], axis: int = 0, drop_duplicates = True, keep = 'first') -> "TSDataset":
+    def concat(cls, tss: List["TSDataset"], axis: Optional[int] = 0, drop_duplicates: Optional[bool]=True, keep: Optional[str]='first') -> "TSDataset":
```

Comment:
Line1966 axis,  drop_duplicates, keep should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional